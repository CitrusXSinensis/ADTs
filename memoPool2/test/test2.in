ra1 1 2 3 5 8 13 21 34 55x
pa
aa 12
ca
pa
fa 1
fa 2
pa
fa 5
ca
pa
fa 54
ca
pa
fa 56

