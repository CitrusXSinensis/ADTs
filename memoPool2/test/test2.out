1 1 2 3 5 8 13 21 34 55
11/16
1 1 2 3 5 8 13 21 34 55 12
Chunk: 1
Chunk: 2
1 3 5 8 13 21 34 55 12
Chunk: 5
8/16
1 3 8 13 21 34 55 12
Chunk: 55
7/16
1 3 8 13 21 34 12
Nullptr
